docking.js

```diff
@@ -1096,6 +1106,25 @@ var DockedDash = GObject.registerClass({
                 this.remove_style_class_name('extended');
             }
         }
+        this._updateVisibleDesktop();
+    }
+
+    _updateVisibleDesktop() {
+        if (!DockManager.settings.get_boolean('intellihide'))
+            return;
+        let monitor = this._monitorIndex;
+        let desktopIconsUsableArea = Me.imports.extension.dockManager.desktopIconsUsableArea;
```

Comment:
```suggestion
        const monitor = this._monitorIndex;
        const desktopIconsUsableArea = Me.imports.extension.dockManager.desktopIconsUsableArea;
```

You could probably avoid the `monitor` one at all.

Reply:
Done.